src/videojs-hls.js

```diff
@@ -973,6 +990,20 @@ videojs.Hls.getMediaIndexByTime = function(playlist, time) {
   return -1;
 };
 
+videojs.Hls.prototype.setCurrentTimeByMediaIndex = function(playlist, mediaIndex) {
```

Comment:
Also, "set" in the name implies some sort of mutation to me. I like side-effect free but the name should change.